rkt/images.go

```diff
@@ -636,6 +713,8 @@ func (f *fetcher) downloadHTTP(url, label string, out writeSyncer, etag string)
 		return nil, fmt.Errorf("bad HTTP status code: %d", res.StatusCode)
```

Comment:
SGTM, but I'm curious if this is an edge case that will ever be hit, since at this point we already verified that range requests were supported via the HEAD request.

Could this happen if the image size changed since the first part was cached, resulting in the requested starting point being larger than the size of the new image?